### ISSUE TITLE
Clouddriver Docker account name support

### ISSUE THREAD

**Issue by nabuskey** [nimakaviani/managed-delivery-k8s-plugin#12]
Currently the plugin assumes there is a docker account named `<ACCOUNT_NAME>-registry`
It would be nice to be able to specify which docker account to look for.

**Comment by nimakaviani** [comment]
Keel resources can only have one account associated with them, and this is a workaround to allow for pulling of associated resources. The following part of the code is where it is happening. 

https://github.com/nimakaviani/managed-delivery-k8s-plugin/blob/6e25bed4542dd31b66eeec0b576db94c48157254/k8s-plugin/src/main/kotlin/com/amazon/spinnaker/keel/k8s/DockerImageResolver.kt#L80-L83

**Comment by JonGilmore** [comment]
Agreed with @nimakaviani - _feels_ more "correct" to make sure that keel just has the right information.

**Comment by nabuskey** [comment]
Opened https://github.com/spinnaker/keel/pull/1999 
Once this is in, we don't need to call clouddriver to get the information we need and no need to use `*-registry` convention.
